job-cleanup/Dockerfile

```diff
@@ -0,0 +1,9 @@
+from alpine:3.5
+
+arg CLIENT_TARBALL=https://github.com/kubernetes-incubator/client-python/archive/master.zip
```

Comment:
should we be pulling from master or a tagged version so we control code changes to the dependency